src/components/ContainerPet/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { nanoid as id } from "nanoid";
+import { useTranslation } from "react-i18next";
+
+// data
+import { animal } from "./CardPet/data";
+
+// component
+import CardPet from "./CardPet";
+
+// styles
+import { StyledContainer } from "./styles";
+
+const ContainerPet = (): JSX.Element => {
+    const { t } = useTranslation();
+
+    return (
+        <StyledContainer>
+            <h1>{t("ContainerCard:h1")}</h1>
```

Comment:
El nombre de la traduccion deberia llamarse como el componente en caso de ser del componente o como la pagina en caso de ser una pagina

```suggestion
            <h1>{t("ContainerPet :h1")}</h1>
```

Reply:
No entiendo estas conversaciones fueron todas marcadas como resolve, pero no veo q se haya modificado nada?